round6/rules.md

```diff
@@ -5,9 +5,9 @@ Each *rule* has an ID number, a title, and a body (a string of text).
 # 2. On Proposals and Turn Order 
 A proposal is a document designated as such by the rules. 
 Each *proposal* has an ID number, a title, and a body (a string of text). 
-If a rule ever lacks a title or has an empty title, a may give it a title at any time.
+If a rule ever lacks a title or has an empty title, it may be assigned a title at any time.
```

Comment:
Anyone can give it a title? I think original intent was for mods, but it was typoed.

Reply:
Cat is correct that was the intent there